executor.go

```diff
@@ -275,6 +294,130 @@ func (e *Executor) provideEventLogger(t *Task) *Logger {
 	}
 }
 
+func (e *Executor) parseTaskFlagsIntoMap(taskName string, flags []string) map[string]FlagArg {
+	taskFlagsMap := make(map[string]FlagArg)
+
+	for _, flag := range flags {
+		var key string
+		var val string
+		splitFlag := strings.Split(flag, "=")
+
+		key, err := e.getVerifiedFlagKey(taskName, splitFlag[0])
+		// If the len(splitFlag) is 1, assume that it is a bool flag
```

Comment:
do we validate this?

Reply:
No explicit validation since we do conditional checks for 2 and >2

I guess it's potentially possible to have len 0 though 🤔